.github/workflows/conventional-pr.yml

```diff
@@ -0,0 +1,43 @@
+name: "Conventional PR"
+
+# Controls when the workflow will run
```

Comment:
I think we don't need comments describing how GitHub workflows work 😄 In my opinion you can remove all comments